cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -839,6 +840,16 @@ Unblock-File $modulePath
 Import-Module -Name $modulePath'.replace('K8S_DIR', ${env:K8S_DIR})
 }
 
+# Setup cni network. This function supports both Docker
+# and containerd.
+function Configure-CniNetworking {
+  if (${env:CONTAINER_RUNTIME} -eq "containerd") {
+    Configure-Containerd-CniNetworking
```

Comment:
Internal functions should be named with underscores, which prevents them from being made visible when this module is imported.

`Configure_Containerd_CniNetworking` etc.

Reply:
Done